packages/react-native-gesture-handler/apple/RNGestureHandler.mm

```diff
@@ -306,6 +313,13 @@ - (void)handleGesture:(UIGestureRecognizer *)recognizer inState:(RNGestureHandle
   [self sendEventsInState:self.state forViewWithTag:tag withExtraData:eventData];
 }
 
+- (RNGestureHandlerEventHandlerType)chooseEventHandlerType
```

Comment:
```suggestion
- (RNGestureHandlerEventHandlerType)eventHandlerType
```

Why not simpler?

Reply:
I was looking for a name that clearly states that it is method, but well... obj-c 😅 Done in [fab60e5](https://github.com/software-mansion/react-native-gesture-handler/pull/3682/commits/fab60e5f5893122a612cfee4f8cde8b2a03789bf)